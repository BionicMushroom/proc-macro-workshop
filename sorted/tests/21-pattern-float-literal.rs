#[sorted::check]
fn main() {
    let x = 0.5;

    #[sorted]
    match x {
        0.5 => println!("0.5"),
        0.1 => println!("0.1"),
        _ => println!("other"),
    }
}
