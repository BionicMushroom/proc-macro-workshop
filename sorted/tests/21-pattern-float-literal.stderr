error: 0.1 should sort before 0.5
 --> tests/21-pattern-float-literal.rs:8:9
  |
8 |         0.1 => println!("0.1"),
  |         ^^^
